viadot/sources/sap_rfc.py

```diff
@@ -84,6 +85,103 @@ def trim_where(where: str) -> Tuple[str, OrderedDictType[str, str]]:
     return where_trimmed, wheres_to_add
 
 
+def detect_extra_rows(
+    row_index: int, data_raw: np.array, chunk: int, fields: List[str]
+) -> Union[int, np.array, bool]:
+    """check if, in between calls to the SAP table, the number of rows have increased.
+        If so, remove the last rows added, to fit the size of the previous columns.
+
+    Args:
+        row_index (int): Number of rows set it down in he first iteration with the SAP table.
+        data_raw (np.array): Array with the data retrieve from SAP table.
+        chunk (int): The part number in which a number of SAP table columns have been split.
+        fields (List[str]): A list with the names of the columns in a chunk.
+
+    Returns:
+        Union[int, np.array, bool]: A tuple with the parameters "row_index", "data_raw", and a new
+            boolean variable "cont" to indicate when the for loop has to be restarted.
+    """
+    cont = False
```

Comment:
The `cont` name is not clear to me. Can you add a different name for this variable if possible? To make the code easier to read.

Reply:
I changed it to `start`, I think it should be fine, but I accept suggestions :)